packages/at_persistence_secondary_server/lib/src/log/commitlog/commit_log_keystore.dart

```diff
@@ -165,12 +166,10 @@ class CommitLogKeyStore
           'Returning the lastSyncedEntry matching regex $regex from cache. lastSyncedKey : ${lastSyncedEntry!.atKey} with commitId ${lastSyncedEntry.commitId}');
       return lastSyncedEntry;
     }
```

Comment:
If there is no entry in _lastSyncedEntryCacheMap, can we just return null rather than iterate through the full commit log? I know this is client-side code so not something to address in this PR but something to follow up on please @sitaram-kalluri (maybe create another ticket for this)

Reply:
Sure Gary, Will create a ticket on this.